autoflake.py

```diff
@@ -1176,6 +1165,35 @@ def merge_configuration_file(args):
         config_path, tail = os.path.split(config_path)
         if not tail:
             break
+    return config
+
+
+def merge_configuration_file(args):
+    """Merge configuration from a file into args."""
+    BOOL_TYPES = {
+        "1": True,
+        "yes": True,
+        "true": True,
+        "on": True,
+        "0": False,
+        "no": False,
+        "false": False,
+        "off": False,
+    }
+
+    if getattr(args, "config_file", None):
```

Comment:
Since this is an argparse Namespace, we don't need getattr:

```suggestion
    if args.config_file:
```

Reply:
Yes, I also wrote that, but the tests did not pass

Because in most tests the `Namespace` is passed without the `config_file` field. 

(Crashes with `AttributeError: 'Namespace' object has no attribute 'config_file'`)

Reply:
Gotcha, I think that's a problem with the tests. I can work on that after merging.